hack/start-verify-ec-task-v2.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/bash
+#
+# Rough hacking guide:
+#
+#  * Commit your changes as required. (No need to push to GitHub.)
+#
+#  * Push a new base image and new task bundles like this:
+#      env MY_QUAY_USER=$USER BUILD_TAG=$(git rev-parse HEAD) ./build-and-push.sh
+#
+#    (Assumes you have the required repos created in quay.io, you're signed in there with podman,
+#    and your quay.io username matches $USER)
+#
+#  * Create a Secret in the current namespace containing the cosign.pub key. This should be a
+#    partial copy of the Secret tekton-chains/signing-secrets. Only the public key should be
+#    included in this Secret. By default the name of the Secret is 'cosign-public-key', set
+#    the PUBLIC_KEY_SECRET environment variable to use a different name. For example:
+#    oc -n tekton-chains get secret signing-secrets -o json | jq '.data."cosign.pub" | @base64d' -r > cosign.pub
+#    oc create secret generic cosign-public-key --from-file=cosign.pub
+#
+#  * Make sure you have an image that has already been signed by Chains.
+#
+#  * Run this script:
+#      ./start-verify-ec-task-v2.sh <image-ref>
+#    where <image-ref> is a valid image reference, e.g. quay.io/spam/bacon@sha256:...
+#
+set -euo pipefail
+
+IMAGE_REF="$1"
+
+PUBLIC_KEY_SECRET="${PUBLIC_KEY_SECRET:-cosign-public-key}"
+
+NAMESPACE="$(oc get sa default -o jsonpath='{.metadata.namespace}')"
```

Comment:
Nit: bit straighter way of getting current namespace

```suggestion
NAMESPACE="$(oc project -q)"
```

Reply:
Nice!